ts/features/idpay/initiative/details/components/TimelineOperationListItem.tsx

```diff
@@ -37,27 +39,53 @@ type TimelineOperationListItemProps = {
   onPress?: () => void;
 };
 
-export const TimelineOperationListItem = (
-  props: TimelineOperationListItemProps
-) => {
-  const { operation, onPress } = props;
+const generateTimelineOperationListItemData = (operation: OperationListDTO) => {
   const hasAmount = "amount" in operation;
 
-  const renderOperationIcon = (operation: OperationListDTO) => {
-    if ("brandLogo" in operation) {
-      return (
-        <Image style={styles.imageSize} source={{ uri: operation.brandLogo }} />
-      );
-    }
+  const OperationIcon = () => {
     switch (operation.operationType) {
       case OnboardingOperationTypeEnum.ONBOARDING:
-        return <Icon name={"bonus"} color="blue" />;
+        return <Icon name={"ok"} />;
+
       case IbanOperationTypeEnum.ADD_IBAN:
-        return <Icon name={"amount"} color="bluegreyLight" />;
+        return <Icon name={"institution"} color="bluegreyLight" />;
+
       case RefundOperationTypeEnum.PAID_REFUND:
-        return <Icon name={"reload"} color="bluegreyLight" />;
+        return <Icon name={"arrowCircleUp"} color="bluegrey" />;
+
+      case OperationTypeEnum.REJECTED_ADD_INSTRUMENT:
+      case OperationTypeEnum.REJECTED_DELETE_INSTRUMENT:
       case RefundOperationTypeEnum.REJECTED_REFUND:
-        return <Icon name={"error"} color="bluegreyLight" />;
+        return <Icon name={"warning"} color="red" />;
+
+      default:
+        if ("brandLogo" in operation) {
+          return (
+            <Image
+              style={styles.imageSize}
+              source={{ uri: operation.brandLogo }}
+            />
```

Comment:
We should start to use the new `IOLogoPaymentType` for payment methods icons. Look at the DS to know how to use them.